### PR TITLE
chore: Update runner image to v2.315.0.

### PR DESCRIPTION
**Pull request by justin-o12**

### CODE REVIEW

**Review by mjnagel — APPROVED** [review]

**Review by mjnagel — CHANGES_REQUESTED** [review]
Need the update here as well - https://github.com/defenseunicorns/uds-package-arc/blob/chore-update-runner/runner-scale-set/values.yaml#L166

File: zarf.yaml [hunk 1]
```diff
@@ -63,6 +63,6 @@ components:
         - "runner-scale-set/values.yaml"
 
   images:
-    - ghcr.io/actions/actions-runner:2.313.0
+    - ghcr.io/actions/actions-runner:2.315.0
     # renovate: datasource=docker
     - ghcr.io/actions/gha-runner-scale-set-controller:0.8.2
```
Comment (bburky) [inline zarf.yaml:68]: Looks like there's a `ghcr.io/actions/gha-runner-scale-set-controller:0.9.0` and `ghcr.io/actions/actions-runner-controller-charts/gha-runner-scale-set-controller:0.9.0` helm chart updates too, but we didn't have Renovate tracking those.

Before doing these updates please test it and review the breaking changes:
https://github.com/actions/actions-runner-controller/releases/tag/gha-runner-scale-set-0.9.0

Also turn on Renovate tracking of this additional image and helm chart please.

If you want to keep the scope to just `ghcr.io/actions/actions-runner:2.315.0` for now you can do this in a separate PR if you want.

Comment (bburky) [reply zarf.yaml:68]: Actually, maybe Renovate is detecting these? They both appear to be on the Reonvate Dashboard https://github.com/defenseunicorns/uds-package-arc/issues/18, but maybe we have Renovate broken.

It mentions a current error:
> WARN: Found renovate config warnings

Comment (justin-o12) [reply zarf.yaml:68]: @bburky and @mjnagel , if possible, I would like to test the charge update to 0.9.0 in a follow-on PR and continue to fight renovate in parallel.  Unless we think we need the updated charts in order to use the latest image.  I'll have to deploy this package locally with my personal GH account to test the runner.  In the meantime, we are blocked on using this is staging with the following error.

```
Runner version v2.313.0 is deprecated and cannot receive messages.
```

I'm hoping an updated image will allow the runners to receive jobs in staging.

**Review by mjnagel — APPROVED** [review]

File: zarf.yaml (hunk 1 above)
Comment (mjnagel) [reply zarf.yaml:68]: I tend to agree - we should be tracking the 0.9.0 update but I don't think they are dependent on each other so moving forward with just the runner image update seems okay.

**Pull request merged**